theme/highcontrast.less

```diff
@@ -818,19 +818,24 @@
 
     /* immersive reader */
     .modals .ui.button.immersive-reader-button,
-    #mainmenu .immersive-reader-button.ui.item:focus {
+    #mainmenu .immersive-reader-button.ui.item:focus,
+    #mainmenu .immersive-reader-button.ui.item:hover {
         background-color: white !important;
         background-image: @immersiveReaderIcon !important;
         background-repeat: no-repeat !important;
-        background-size: contain !important;
+        background-size: 80% !important;
         background-position: center!important;
+        filter: none!important
     }
 
     // skillmap embedded tutorials
     .tutorial.tutorial-embed {
         .immersive-reader-button.ui.item,
-        .immersive-reader-button.ui.item:focus {
-            background-size: 1.7rem;
+        .immersive-reader-button.ui.item:focus,
+        .immersive-reader-button.ui.item:hover {
+            background-size: 80%!important;
```

Comment:
For high contrast to match the other buttons it might be better to do white outline / black background / white version of the icon I suppose (but still LGTM)

![image](https://user-images.githubusercontent.com/5615930/112561095-c0d6a980-8d91-11eb-925a-91461b9add9a.png)